db/seeds.d/112-upgrade_tasks.rb

```diff
@@ -0,0 +1,6 @@
+UpgradeTask.define_tasks(:foreman) do
+  [
+    {:name => 'purge:foreman_docker'},
+    {:name => 'purge:trends'}
```

Comment:
rubocop wants coma here
```suggestion
    {:name => 'purge:trends'},
```